src/extrudeText.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * Based on https://3d.hrg.hr/jscad/jscad_text/
+ */
+
+import { extrusions, geometries, maths, booleans } from "@jscad/modeling";
+import { textToPaths, TextToPathsOptions } from "./textToPaths";
+
+const { extrudeLinear } = extrusions;
+const { subtract, union } = booleans;
+
+// geometry or points array
+const isCCW = (points: maths.vec2.Vec2[]) => {
```

Comment:
Basically, this is the same as maths.area() so let's use area()

https://www.openjscad.xyz/docs/module-modeling_maths_utils.html#.area